site/_data/docs/extensions/toc.yml

```diff
@@ -34,6 +34,7 @@
     - url: /docs/extensions/mv3/background_pages
     - url: /docs/extensions/mv3/match_patterns
     - url: /docs/extensions/mv3/promises
+    - url: /docs/extensions/mv3/cross-origin-isolation
```

Comment:
general comment (no action needed): We don't seem to be consistent with using hyphens or underscores as word separators. We should probably settle on a style.